snapshots/types/format.go

```diff
@@ -3,4 +3,7 @@ package types
 // CurrentFormat is the currently used format for snapshots. Snapshots using the same format
 // must be identical across all nodes for a given height, so this must be bumped when the binary
 // snapshot output changes.
-const CurrentFormat uint32 = 1
+// Format 1 was the original format.
+// Format 2 serailizes app version in addition to the original 
```

Comment:
Asking to learn: does `format` indicate the format of the snapshot itself(with format2 indicating the current updated version with the app version?

Reply:
Yes, that's correct. Every time we change how snapshots are serialized, we should increase this `CurrentFormat`.

This is done to ensure that nodes are capable of interpreting snapshot chunks received from other nodes. If node A supports format 1 and receives a chunk from node b that supports format 2, node A can immediately reject it. Otherwise, it wouldn't know how to deserialize it.

I changed the godoc a bit to make it clearer